.gitlab-ci.yml

```diff
@@ -28,11 +28,14 @@ build-in-docker:
     # Check that the AutoGenerated files are correct by checking git status
     - git update-index -q --refresh
     - git diff-index --exit-code HEAD
-
   artifacts:
     paths:
       - build/install
 
+build-in-docker-alpine:
+  extends: build-in-docker
+  image: prplfoundationinc/prplmesh-builder:alpine-3.11.3
```

Comment:
It would be nice to build the docker image as part of the pipeline as well. But that's again something we can add later.